utils/src/main/java/com/indix/gocd/utils/store/S3ArtifactStore.java

```diff
@@ -187,4 +199,13 @@ public RevisionStatus getLatest(AmazonS3Client client, Artifact artifact) {
         }
         return null;
     }
+
+    public static AmazonS3 getS3client(GoEnvironment env) {
+        AmazonS3ClientBuilder amazonS3ClientBuilder = AmazonS3ClientBuilder.standard();
```

Comment:
@manojlds I'm not aware of the latest client authentication chain. But does it cover all the things as [specified here](https://github.com/indix/gocd-s3-artifacts/pull/41/files#diff-e36fde5bc02a23e18c6b99a90ab85c63R40)?